clone-and-checkout.sh

```diff
@@ -39,15 +39,14 @@ if [[ -z ${DPEBOT_GITHUB_TOKEN} ]] ; then
   exit 1
 fi
 
-BRANCH="repositorygardener$(date +"%Y%m%d")b${RANDOM}"
+BRANCH="dpebot-repositorygardener"
```

Comment:
It'd be nice to be able to set this from the calling script - we may want to automate PRs for more than just updating deps.